### ISSUE TITLE
cdnizer fails the security audit

### ISSUE THREAD

**Issue by Kjir** [OverZealous/cdnizer#31]
When installing cdnizer 2.0.0 (through another npm package, but that's not relevant), you get the following warning:

```
warning webpack-s3-plugin > cdnizer > nomnom@1.8.1: Package no longer supported. Contact support@npmjs.com for more info.
```

Also the security audit now includes two issues:
![image](https://user-images.githubusercontent.com/64094/49081345-df224e00-f246-11e8-8fae-82a2754d7d8e.png)

Here's the links to the advisories:
 * https://nodesecurity.io/advisories/31
 * https://nodesecurity.io/advisories/577

**Comment by OverZealous** [comment]
Thanks for the report. I can't fix the semver error inside the jsdelivr-cdn-data, but I don't think it should matter.

I didn't write the CLI code for the library, and I don't feel like maintaining it any longer, so I'm stripping it from the next version.

I've published the fixes as v3.0.0

**Comment by OverZealous** [comment]
Actually, I have to republish it as v3.0.1. Stupid CLI.

**Comment by Kjir** [comment]
Thank you for taking care of this, and in such a short time!

**Comment by Kjir** [comment]
BTW: it looks like `jsdelivr-cdn-data` has an updated version that should fix the issue with semver. Updating the dependency should get rid of that problem too

**Comment by OverZealous** [comment]
Thanks for the heads up!

@noqcks provided a PR to fix this in #33. It's been published as 3.0.2 for `cdnizer` and 2.0.2 for `gulp-cdnizer`.
